### DISCUSSION TITLE
Sir please it's urgent i want contact you please email me on this I'd mnothingbutacomedy@gmail.com

### DISCUSSION THREAD

**Discussion by Sinistervibranium** [TermuxHackz/anonphisher#19] [Show and tell]
Sir please it's urgent i want to contact you please email me on this I'd mnothingbutacomedy@gmail.com

**Comment by TermuxHackz** [comment]
Hi there @Sinistervibranium for any help you may needed please contact me [by email](mailto:AnonyminHack5@protonmail.com)

**Comment by Sinistervibranium** [comment]
I'm make some GitHub repository but I can't install in termux I'm tried
bash.sh and all but not working please talk me How can I do that

**Reply by TermuxHackz** [reply]
Hi there @Sinistervibranium please send me a detailed screenshot of the problem you are experiencing
